memdown.js

```diff
@@ -46,85 +44,66 @@ function MemIterator (db, options) {
     this._start = ltgt.lowerBound(options)
     this._end = ltgt.upperBound(options)
 
-    if (typeof this._start === 'undefined')
-      this._tree = tree.begin
-    else if (ltgt.lowerBoundInclusive(options))
-      this._tree = tree.ge(this._start)
-    else
-      this._tree = tree.gt(this._start)
+    if (typeof this._start === 'undefined') this._tree = tree.begin
+    else if (ltgt.lowerBoundInclusive(options)) { this._tree = tree.ge(this._start) } else this._tree = tree.gt(this._start)
 
     if (this._end) {
-      if (ltgt.upperBoundInclusive(options))
-        this._test = lte
-      else
-        this._test = lt
+      if (ltgt.upperBoundInclusive(options)) this._test = lte
+      else this._test = lt
     }
-
   } else {
     this._incr = 'prev'
     this._start = ltgt.upperBound(options)
     this._end = ltgt.lowerBound(options)
 
-    if (typeof this._start === 'undefined')
-      this._tree = tree.end
-    else if (ltgt.upperBoundInclusive(options))
-      this._tree = tree.le(this._start)
-    else
-      this._tree = tree.lt(this._start)
+    if (typeof this._start === 'undefined') this._tree = tree.end
+    else if (ltgt.upperBoundInclusive(options)) { this._tree = tree.le(this._start) } else this._tree = tree.lt(this._start)
 
     if (this._end) {
-      if (ltgt.lowerBoundInclusive(options))
-        this._test = gte
-      else
-        this._test = gt
+      if (ltgt.lowerBoundInclusive(options)) this._test = gte
+      else this._test = gt
```

Comment:
Spontaneous comment not related to this change: This looks like it could be simplified. It's basically the same code wrapped in two if branches.